build.sh

```diff
@@ -72,25 +81,32 @@ function check_ut_control_revision()
 
 # Check if the common document configuration is present, if not clone it
 pushd ${FRAMEWORK_DIR} > /dev/null
+
+configure_ut_control() {
+    cd ./ut-control
+    #git checkout ${UT_CONTROL_PROJECT_VERSION}
+    git checkout feature/gh38-separate-obj-dir-ut-control
+    TARGET=${TARGET} ./configure.sh
```

Comment:
`TARGET=${TARGET};./configure.sh` don't you need a semicolon?

Reply:
it didn't work with semicolon.

Reply:
`TARGET=${TARGET} ./configure.sh`

should now be passing as `$1`

`./configure.sh ${TARGET}`